examples/pipelines/commoncrawl/components/download_commoncrawl_segments/src/main.py

```diff
@@ -0,0 +1,77 @@
+import logging
+from typing import List
+
+import io
+import boto3
+
+import dask.bag as db
+import dask.dataframe as dd
+from dask.delayed import delayed
+
+import pandas as pd
+
+import requests
+from warcio.archiveiterator import ArchiveIterator
+
+from fondant.component import DaskTransformComponent
+
+logger = logging.getLogger(__name__)
+
+BASE_URL = "https://data.commoncrawl.org/"
+
+
+def get_records_from_warc_file(warc_file: str, n_records_to_download: int) -> List:
```

Comment:
The `n_records_to_download` should be an optional argument and below you should only break if it not `None`